src/components/home/Landing.tsx

```diff
@@ -1,8 +1,15 @@
+import Image from "next/image";
+import banner from "@/public/home/landing.webp";
+
 const Landing = () => {
   return (
-    <div className="flex w-full items-center justify-center text-3xl">
-      {" "}
-      Landing
+    <div className="relative mt-[5.8%] flex flex-row items-center justify-center text-[115px]">
```

Comment:
I recommend to use a tailwind class instead of hard coding the pixels for text.
For example, `text-xl` or `text-4xl` etc